### PR TITLE
Add open-cluster-management-policies namespace to the managed and hub cluster

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
Add open-cluster-management-policies namespace to the managed and hub cluster

After https://github.com/stolostron/governance-policy-framework-addon/pull/516 merged, will add test cases.

Ref: https://issues.redhat.com/browse/ACM-13609

### CODE REVIEW

File: test/e2e/case1_framework_deployment_test.go [hunk 1]
```diff
@@ -72,7 +73,7 @@ var _ = Describe("Test framework deployment", Ordered, func() {
 
 			By(logPrefix +
 				"removing the framework deployment when the ManagedClusterAddOn CR is removed")
-			Kubectl("delete", "-n", cluster.clusterName, "-f", case1ManagedClusterAddOnCR, "--timeout=90s")
+			Kubectl("delete", "-n", cluster.clusterName, "-f", case1ManagedClusterAddOnCR, "--timeout=180s")
```
Comment (yiraeChristineKim) [inline test/e2e/case1_framework_deployment_test.go:76]: flaky test issue

**Review by mprahl — APPROVED** [review]

**Review by mprahl — CHANGES_REQUESTED** [review]
This PR needs to be in OCM-io.

**Comment by mprahl** [comment]
Closing this one since it's covered in #684

**Pull request closed without merging**